azure-pipelines.yml

```
@@ -46,6 +46,15 @@ jobs:
           PYTHON_ARCH: 'x64'
           NPY_USE_BLAS_ILP64: 1
           BITS: 64
+        py_3.10_32:
+          PYTHON_VERSION: "3.10-dev"
```

If it's not available yet on Azure, we should still be able to build wheels with the following no?
```suggestion
          AZURE_PYTHON_VERSION: "3.9"
          PYTHON_VERSION: "3.10"
```
```suggestion
          PYTHON_VERSION: "3.10-dev"
```